docs/development-guidelines/global-deployment.md

```diff
@@ -1,18 +1,32 @@
 # Global Deployment
 
-This document outlines a workaround for a known upstream bug handling global-standard type Azure OpenAI deployments (issue of resource deletion conflicts).
+This document outlines the migration guidance for transitioning to a global deployment SKU for GPT models in Azure OpenAI, as well as troubleshooting steps for handling resource deletion conflicts.
 
-## Issue
+## Migration Guidance
+
+To migrate to a global deployment SKU, follow these steps:
+
+1. **Add a Parameter for DeploymentSkuName**: Update your deployment configuration to include a parameter for `DeploymentSkuName`.
```

Comment:
better to specify add it to where

Reply:
Yes, please specify which file and add example with code

Reply:
This is how I do it, but it may vary per template:

1) Add a parameter for each model used in the infrastructure to set the sku name, like chatDeploymentSkuName. Use that parameter as the sku input for the OpenAI deployments.
   
   param chatDeploymentSkuName string = ''
   
2) Add a parameter mapping to main.parameters.json or main.bicepparam that maps an azd environment variable to that parameter. This allows developers to customize the sku per azd environment easily. In the parameter mapping, set the default to GlobalStandard.
    
   ````
    "chatDeploymentSkuName": {
      "value": "${CHAT_DEPLOYMENT_SKU_NAME=GlobalStandard}"
    }
   ````

   param chatDeploymentSkuName = readEnvironmentVariable('CHAT_DEPLOYMENT_SKU_NAME', 'GlobalStandard'))
  ````

4) In the documentation somewhere, provide guidance about how to set the sku back to standard, for developers that need it:
   
   ```shell
   azd env set CHAT_DEPLOYMENT_SKU_NAME Standard
   ```

Reply:
Updated. Thanks for suggestions.